<div class="chatlog__message-group">
  <div id="chatlog__message-container-447701637029691392" class="chatlog__message-container " data-message-id="447701637029691392">
    <div class="chatlog__message">
      <div class="chatlog__message-aside">
        <img class="chatlog__avatar" src="https://cdn.discordapp.com/avatars/127674083331538944/4893b43ea9ee2231b8ffb8b05c305ce3.png?size=512" alt="Avatar" loading="lazy">
      </div>
      <div class="chatlog__message-primary">
        <div class="chatlog__header">
          <span class="chatlog__author" style="color: rgb(155, 89, 182)" title="norio_nomura#8334" data-user-id="127674083331538944">norio_nomura</span>
          <a href="/channels/430242233468452865?category=main&amp;channel=swiftbot-sandbox&amp;message_id=447701637029691392"><span class="chatlog__timestamp">20-May-18 10:06 AM</span></a>
        </div>
        <div class="chatlog__content chatlog__markdown">
          <span class="chatlog__markdown-preserve">Vapor3が依存するSwiftNIOも。 <span class="chatlog__markdown-mention" title="swift41#2144">@swift-4.1.3</span>
            <div class="chatlog__markdown-pre chatlog__markdown-pre--multiline language-swift">import NIO struct E: Error {} let eventLoop = EmbeddedEventLoop() let promises: [EventLoopPromise&lt;Void&gt;] = (0..&lt;100).map { (_: Int) in eventLoop.newPromise() } let futures = promises.map { $0.futureResult } let fN: EventLoopFuture&lt;Void&gt; = EventLoopFuture&lt;Void&gt;.andAll(futures, eventLoop: eventLoop) _ = promises.map { $0.fail(error: E()) } do { () = try fN.wait() print("should've thrown an error") } catch _ as E { print("good") } catch let e { print("error of wrong type \(e)") }</div>
          </span>
        </div>
      </div>
    </div>
  </div>
</div>